onnx/examples/make_model.ipynb

```diff
@@ -9,7 +9,6 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "The ir_version in model: 7\n",
       "\n",
```

Comment:
I think L13 needs to be removed as well. The check here is quite unfriendly for people who want to update it... I will try to improve it in another PR. Thank you for the effort.